content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -108,32 +119,40 @@ cluster for mesh expansion, run the following commands on a machine with cluster
           -o jsonpath='{.data.cert-chain\.pem}' |base64 --decode > cert-chain.pem
     {{< /text >}}
 
-### Setting up the machines
+### Setting up the VM
 
 Next, run the following commands on each machine that you want to add to the mesh:
 
-1.  Install the Debian package with the Envoy sidecar:
+<!-- TODO: find a better way to reference release in the URL instead of manually sync... -->
+
+1.  Copy the `cluster.env` and `*.pem` files that you created in the previous section to the VM. For example:
 
     {{< text bash >}}
-    $ curl -L https://storage.googleapis.com/istio-release/releases/1.0.0/deb/istio-sidecar.deb > istio-sidecar.deb
-    $ dpkg -i istio-sidecar.deb
+    $ export GCE_NAME="your-gce-instance"
+    $ gcloud compute scp --project=${MY_PROJECT} --zone={MY_ZONE} {key.pem,cert-chain.pem,cluster.env,root-cert.pem} ${GCE_NAME}:~
     {{< /text >}}
 
-1.  Copy the `cluster.env` and `*.pem` files that you created in the previous section to the VM.
+1.  Install the Debian package with the Envoy sidecar.
+
+    {{< text bash >}}
+    $ gcloud compute ssh "${GCE_NAME}"
+    $ curl -L https://storage.googleapis.com/istio-release/releases/1.1.0-rc.0/deb/istio-sidecar.deb > istio-sidecar.deb
+    $ sudo dpkg -i istio-sidecar.deb
+    {{< /text >}}
 
 1.  Add the IP address of the Istio gateway (which we found in the [previous section](#preparing-the-kubernetes-cluster-for-expansion))
```

Comment:
Can we move the command to get gateway ip to this section so people don't have to jump around? Considering gateway ip is not used in previous section.

Reply:
You mean in `Set up the VM` section and step 1?